libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/impl/EmailNotificationService.java

```diff
@@ -251,20 +251,21 @@ void processExpiryEntry(StringBuilder body, final String entryNames, final Strin
 
     boolean sendEmail(Set<String> recipients, String subject, String body) {
         final AtomicInteger counter = new AtomicInteger();
-        boolean status = true;
         // SES imposes a limit of 50 recipients. So we convert the recipients into batches
         if (recipients.size() > SES_RECIPIENTS_LIMIT_PER_MESSAGE) {
             final Collection<List<String>> recipientsBatch = recipients.stream()
                     .collect(Collectors.groupingBy(it -> counter.getAndIncrement() / SES_RECIPIENTS_LIMIT_PER_MESSAGE))
                     .values();
+            boolean status = true;
             for (List<String> recipientsSegment : recipientsBatch) {
-                status = sendEmailMIME(subject, body, status, recipientsSegment);
```

Comment:
As we talked about on a different pull request, the status will not be part of the interface